src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -764,6 +764,10 @@ def get_action_list(
         ]
 
     def post(self, request: Request) -> Response:
+        logger.info(
+            "slack.action.post",
+            extra=request.data,
```

Comment:
will this also log their IP? This would help us get the range that SRE can then help double check for us

Reply:
i'm not sure.. request.data is a `dict[str, Any]`

Reply:
looks like it might be part of meta? https://stackoverflow.com/questions/4581789/how-do-i-get-user-ip-address-in-django

Reply:
updated :eyes:

Reply:
would this break is META is missing or empty?

Reply:
is META ever missing? it's initialized to an empty dict in the django `HTTPRequest` class

Reply:
i can do getattr